### PR TITLE
Revert "Create white glove AB test (#42783)"

### PR DESCRIPTION
**Pull request by southp**
### Changes proposed in this Pull Request

Since we have decided to sunset the Expert Guidance upsell experiment, this PR attempts to revert all the changes introduced by #42783. Since a direct revert for f9bd012961f28a49646139bf97452c3ed41e59eb is unfortunately not possible, I'm implementing it in two folds:

1. Create a revert commit by the command `git revert f9bd012961f28a49646139bf97452c3ed41e59eb`
1. Keep all the reverting changes in this base branch, and resolve the conflicts in a follow-up PR(#44024 ) so we can see if the conflict resolution make sense easier.

#### Testing instructions

Everything in this PR is simply from `git revert`, so we don't really have to pay much attention here except for the conflict resolution part of #44024.

### CODE REVIEW

File: client/lib/abtest/active-tests.js [hunk 1]
```diff
@@ -138,6 +138,7 @@ export default {
 		localeTargets: 'any',
 		allowExistingUsers: true,
 	},
+<<<<<<< HEAD
```
Comment (belcherj) [inline client/lib/abtest/active-tests.js:141]: You have unfixed merge conflicts.

**Comment by southp** [comment]
@belcherj Thanks for driving by. As noted in the description, I kept those conflicts by intention. 
The reason is that a direct revert to the original PR is no longer possible, but removing them piece by piece is also unnecessarily daunting. Thus, the strategy here is to make the best use of the revert commit, and make the conflict resolution visible enough so that we can focus on only the conflicting part.
  
That's why this PR isn't labeled as Needs Review, with all the conflict resolution implemented in https://github.com/Automattic/wp-calypso/pull/44024, which is based on this PR.

Does this make sense to you?

File: client/lib/abtest/active-tests.js (hunk 1 above)
Comment (southp) [reply client/lib/abtest/active-tests.js]: Now that the conflict resolving branch has been merged, I am marking this as resolved.

**Comment by southp** [comment]
I am going to deploy the expert guidance rollback as it is. The conflict resolution part has been reviewed, and the rest are just mechanical changeset generated by git command, and it can be reverted easily. In all I don’t think it worths the time of extensive QA.

**Comment by belcherj** [comment]
Hey, @southp sorry I should have read closer. I was on making sure the desktop builds were passing duty. My apologies.

**Pull request merged**